addons/mail/static/src/messaging.js

```diff
@@ -263,6 +273,15 @@ export class Messaging {
                         this.createMessage(body, message, thread);
                     }
                     break;
+                case "mail.record/insert":
+                    {
+                        const partners_in_payload = notif.payload.Partner;
+                        for (const partner in partners_in_payload) {
+                            this.partners[partners_in_payload[partner].id].im_status =
+                                partners_in_payload[partner].im_status;
+                        }
```

Comment:
Would be easier to read these LOCs with a more brief local variable. Even just `data` is good (it's local variable anyway, so context is easy to grasp)